### PR TITLE
[16.0][IMP] sale_planner_calendar: Add planner events as activities to manage notifications

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa TT51632
ping @sergio-teruel @CarlosRoca13

### CODE REVIEW

File: sale_planner_calendar/models/res_users.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    def _get_sale_planner_calendar_events_domain(self):
+        return [
+            ("user_id", "=", self.env.user.id),
+            ("target_partner_id", "!=", False),
+            ("sale_planner_state", "=", "pending"),
+        ]
+
+    @api.model
+    def get_action_sale_planner_calendar_event(self):
+        return self.env["ir.actions.act_window"]._for_xml_id(
```
Comment (sergio-teruel) [inline sale_planner_calendar/models/res_users.py:23]: Maybe would be interest return by default de kanban view...

**Review by sergio-teruel — COMMENTED** [review]
Minor comment

**Review by sergio-teruel — CHANGES_REQUESTED** [review]
Minor comment

File: sale_planner_calendar/models/res_users.py (hunk 1 above)
Comment (carlosdauden) [reply sale_planner_calendar/models/res_users.py:23]: Kanban is now the default view for this action

**Review by sergio-teruel — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**